### ISSUE TITLE
Apple Watch

### ISSUE THREAD

**Issue by rovo79** [intelligenate/dfc#22]
Is the intention that my watch should be paired to the DFC device, or device running the cycling app(eg zwift)?

Currently it looks to be paired to DFC. 

I cant find a way to maintain an HR data stream to zwift.

**Comment by wcotis** [comment]
Try the Echo app on your iPhone https://www.echoheartrate.app/

**Comment by veneman** [comment]
DFC can pair with both an Apple Watch and Zwift at the same time and send power and cadence data. For broadcasting heart rate you'll have to check out other resources such as the Echo app mentioned here.

**Comment by rovo79** [comment]
I was able to sort it out, and get my heart rate sent to Zwift (on iPad) reliably.

DFC broadcasting from Peloton to iPad running Zwift. 
Zwift Companion on Apple Watch sending heart data to Zwift Companion on iPhone, iPhone sending to iPad. 

I finally realized they all need to be on the same wifi SSID to function and stay connected in Zwift. I have several SSIDs in my house that the devices roam across. Forcing them all on the same, makes it work.
